### PR TITLE
Add some commands to make web development easier

### PR DESCRIPTION
**Pull request by RoboErikG**
bgit fclone web|android|ios|devtools - checks out and configures your personal fork of blockly
bgit fupdate - updates the master and develop branches of your fork
bgit fbranch <branch> - create a new branch on your fork based off develop

---
This change is [Reviewable](https://reviewable.io/reviews/google/blockly-tools/7)

### CODE REVIEW

**Comment by RoboErikG** [comment]
PTAL

File: bgit [hunk 1]
```diff
@@ -114,6 +114,124 @@ function run_lint() {
...
+    fork_repo_ext="-devtools"
+  elif [ "$fork_project" != "web" ]; then
+    # 'web' is a valid value, but we don't do anything for it
+    # which is why we just check it at the end.
+    echo "Must specify one of 'web', 'android', 'ios', or 'devtools' to clone a fork of."
+    return;
+  fi
+
+  fork_repo="$fork_repo_base$fork_repo_ext"
+  fork_local_dir="$BLOCKLY_ROOT/${fork_project}fork"
```
Comment (AnmAtAnm) [inline bgit]: fork -> $BLOCKLY_USER

Comment (RoboErikG) [inline bgit]: done.

File: bgit [hunk 2]
```diff
@@ -114,6 +114,124 @@ function run_lint() {
...
+# Verifies that you are currently in a forked git project with
+# no uncommitted changes
+function verify_fork() {
+  if [ "$BLOCKLY_INIT" != true ]; then
+    echo "Must run envsetup.sh before updating your web fork."
+    return 1
+  fi
+
+  # Check that we're in a forked repo
+  if [ $(git remote -v | grep -i origin.*$BLOCKLY_USER | wc -l) = "0" ]; then
```
Comment (AnmAtAnm) [inline bgit]: TODO: Allow user to override $BLOCKLY_USER when cloning and verifying the fork

**Pull request merged**